routers/api/v1/repo/commits.go

```diff
@@ -325,11 +325,11 @@ func DownloadCommitDiffOrPatch(ctx *context.APIContext) {
 	}
 }
 
-// GetCommitPullRequest returns the pull request of the commit
+// GetCommitPullRequest returns the merged pull request of the commit
 func GetCommitPullRequest(ctx *context.APIContext) {
```

Comment:
It is at the wrong place, because other handlers all support branch/tag (ResolveRefOrSha), but this one doesn't

Reply:
I moved the handler to `/{sha}`, see fd35dc9e5cbf47eef0494661aef07a929ad4092f